### DISCUSSION TITLE
Runtime error?

### DISCUSSION THREAD

**Discussion by webyrd** [ekzhang/percival#6] [Q&A] [answered]
Hi Eric!

I'm getting an error for the plotting example, under the latest macOS (12.0.1):

Firefox:

`Runtime Error: TypeError: l5 is not a constructor`

Chrome:

`Runtime Error: Uncaught TypeError: l5 is not a constructor`

Safari:

`Runtime Error: TypeError: undefined is not a constructor (evaluating 'new l5(this)')`

Cheers,

--Will

**Comment by ekzhang** [comment, marked as answer]
Thanks Will! This is an upstream issue with [Rollup](https://github.com/rollup/rollup) unfortunately. It's a bit tricky to debug, but I'm trying to find a workaround for this.

**Reply by ekzhang** [reply]
Just fixed in ba5b2ca1b936c75665ec0ac08385ea79b20af395 - you should be able to try it out now!

Edit: Just deployed to prod, had to resolve a caching issue. It's live now at https://percival.ink/ :)

![image](https://user-images.githubusercontent.com/7550632/144116046-fd0fc224-eb23-4854-9b6e-76360780b80b.png)
